library/book.rb

```diff
@@ -0,0 +1,15 @@
+require './rental'
```

Comment:
**- [ ] Like mentioned in the main comment, please complete your readme file so that it reflects the correct documentation of your project with all the relevant sections and make your project professional. like the previous CR mentioned.**